### ISSUE TITLE
Example examples/xml-file.js does not work

### ISSUE THREAD

**Issue by highsource** [derhuerst/parse-gml-polygon#6]
```
parse-gml-polygon-upstream\examples>node xml-file.js
parse-gml-polygon-upstream\index.js:150
        if (!exterior) throw new Error('invalid ' + _.name + ' element')
                       ^

Error: invalid gml:Polygon element
    at parsePolygonOrRectangle (parse-gml-polygon-upstream\index.js:150:23)
    at parse (parse-gml-polygon-upstream\index.js:228:17)
    at EventEmitter.parser.once (parse-gml-polygon-upstream\examples\xml-file.js:18:17)
    at EventEmitter.emit (parse-gml-polygon-upstream\node_modules\eventemitter3\index.js:129:35)
    at EventEmitter.handleLexerData (parse-gml-polygon-upstream\node_modules\xml-reader\dist\reader.js:72:28)
    at EventEmitter.emit (parse-gml-polygon-upstream\node_modules\eventemitter3\index.js:129:35)
    at emit (parse-gml-polygon-upstream\node_modules\xml-lexer\dist\lexer.js:84:15)
    at parse-gml-polygon-upstream\node_modules\xml-lexer\dist\lexer.js:118:13
    at step (parse-gml-polygon-upstream\node_modules\xml-lexer\dist\lexer.js:234:9)
    at EventEmitter.lexer.write [as parse] (parse-gml-polygon-upstream\node_modules\xml-lexer\dist\lexer.js:240:13)
```

**Comment by derhuerst** [comment]
I think this is caused by pladaria/xml-reader#9.

**Comment by highsource** [comment]
Let's remove `stream:true` for a while?

**Comment by derhuerst** [comment]
> Let's remove `stream:true` for a while?

Although I'm really inclined to remove the example (the world doesn't need more shitty code that freezes or even crashes with 100mb of GML), it is a relevant use case that people want to parse a GML file. I will adapt it heavy-heartedly. 😢

**Comment by derhuerst** [comment]
Fixed by 362e55b.
